### ISSUE TITLE
Embedded Player Modifications

### ISSUE THREAD

**Issue by amavarick** [AzuraCast/AzuraCast#448]
Docker Installation
Host OS: Ubuntu 16.04.3 LTS

I really love the simplicity of the embedded player and also the public page.  I searched the closed tickets to see if there was any other request for this before asking you.

Below is the embedded player on my primary radio site.  Below that is the public radio player on the Azuracast site.  It would be great to have the embedded player to match the public page player.

1.  Would you be willing to setup an iframe embedded player that looks identical to the public player that includes the album art, current songtime/total song time, Song History, Request Song and Playlist options?

2.  If that is not something that aligns with your goals for the application, would you consider adding the album artwork to the embedded player and maybe current songtime/total song time?

Embedded on my music website:
![image](https://user-images.githubusercontent.com/20918832/36633844-e7a6b086-1969-11e8-84a6-6226a7773ea7.png)

The Azuracast Public Page:
![image](https://user-images.githubusercontent.com/20918832/36633858-2af7b146-196a-11e8-8987-4dcb69566acd.png)

**Comment by SlvrEagle23** [comment]
@amavarick I'd be happy to unify the styling of the fully-designed and embeddable public pages. It would save me some duplicated efforts on the code end, so there's no reason not to.

@ckeurk The idea with the thumbnail is to eventually have it open directly inside the window, in a magnified "zoom" viewer as you'd expect. I'll add that as part of this enhancement, since alongside it you'll be able to style the zoomed thumbnail image.

**Comment by dreinn** [comment]
Hello,
since public page player is not accessible for visual impaired, and
also the embedded player, should I open a new ticket about it?
It is not possible to find the button!

2018-02-25 0:36 GMT+01:00, SlvrEagle23 <notifications@github.com>:
> @amavarick I'd be happy to unify the styling of the fully-designed and
> embeddable public pages. It would save me some duplicated efforts on the
> code end, so there's no reason not to.
>
> @ckeurk The idea with the thumbnail is to eventually have it open directly
> inside the window, in a magnified "zoom" viewer as you'd expect. I'll add
> that as part of this enhancement, since alongside it you'll be able to style
> the zoomed thumbnail image.
>
> --
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/AzuraCast/AzuraCast/issues/448#issuecomment-368269123


-- 
Simone Dal Maso
www.nvapple.it
www.nvda.it

**Comment by SlvrEagle23** [comment]
@amavarick The embeddable public page and full-size public page have been updated to both feature the same rich metadata support, including album artwork and playback time.

@dreinn Along with this update, I have also made sure to apply the `role="button"` and a title attribute to the play button, which should hopefully help with this issue.

**Comment by amavarick** [comment]
Awesome, I can see the album artwork and the playback time and everything seems to update properly.

However, the music doesn't play after pressing the play button.

![image](https://user-images.githubusercontent.com/20918832/36686607-ef50fe68-1af4-11e8-8955-ff5f15ed5ebe.png)

![image](https://user-images.githubusercontent.com/20918832/36686582-dfc8f554-1af4-11e8-902c-b2e826e00a7e.png)

**However checking a web player on the same site, volume and sound is fine.  After looking at debugger, as soon as I press the play button, the following occurs.  Starts with, "Invalid URI load of media resource failed"**

![image](https://user-images.githubusercontent.com/20918832/36686503-a34ae1be-1af4-11e8-8d1c-e1140a4c051e.png)

**Comment by SlvrEagle23** [comment]
@amavarick This should be fixed now.

**Comment by dreinn** [comment]
first of all thank you because now the play/pause is seen also for
visual impaired.
Also for me, music is not audible.
I can listen if I connect to the 8004 port via icecast, but not with
your player.


2018-02-26 22:03 GMT+01:00, SlvrEagle23 <notifications@github.com>:
> @amavarick This should be fixed now.
>
> --
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/AzuraCast/AzuraCast/issues/448#issuecomment-368649454


-- 
Simone Dal Maso
www.nvapple.it
www.nvda.it

**Comment by amavarick** [comment]
Thank you, it looks great and works well.

**Comment by SlvrEagle23** [comment]
@dreinn What audio format are you using? Some browsers won't support direct streaming of some audio formats, which is why the public radio site may not work but a dedicated player, like VLC, will.

**Comment by reans** [comment]
is it possible to make the embed-player autoplay?

**Comment by SlvrEagle23** [comment]
@reans Let's add that as a new enhancement. Closing this issue as wrapped up.
